lib/symbols.js

```diff
@@ -13,7 +13,13 @@ const keys = {
   kMiddlewares: Symbol('fastify.middlewares'),
   kCanSetNotFoundHandler: Symbol('fastify.canSetNotFoundHandler'),
   kFourOhFourLevelInstance: Symbol('fastify.404LogLevelInstance'),
-  kFourOhFourContext: Symbol('fastify.404ContextKey')
+  kFourOhFourContext: Symbol('fastify.404ContextKey'),
+  kReplySerializer: Symbol('reply.Serializer'),
```

Comment:
`reply.Serializer` -> `reply.serializer` final nit from me then LGTM :) apologies for being a pain with my partial reviews

Reply:
Don't worry! jaja

Reply:
@cemremengu Could you check it again? I already fixed it